docs/configuration-reference.md

```diff
@@ -2104,6 +2131,23 @@ prometheus_remote_write:
   - [<remote_write>]
 ```
 
+### client_tls_config
+
+The `client_tls_config` block configures the client for tls in conjunction with the `server_tls_config`.
+Values need to be defined if the server is requesting a certificate 
+(Client Auth Type = RequireAndVerifyClientCert || RequireAnyClientCert). 
+
+```yaml
+# File path for the client cert
+[cert_file: <string>]
+  
+# File path for the client key
+[key_file: <string>]
+  
+# File path to client CA certificate
+[ca_file: <string>]
+```
+
```

Comment:
This is already documented by `tls_config`. I also think we can move the warning about requiring the settings to the `integrations.http_tls_config` field's comment.

Reply:
Good idea and done